docs/lab/overview/introduction.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introduction"
+---
+
+Vital's lab test API allows digital health companies to carry out at-home lab testing.
+Companies can use the API to order a variety of tests, using various sampling methods.
```

Comment:
various -> multiple. In the first part of the sentence the word `variety` is used. Using a synonymous later makes the reading more pleasant